### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by tmccaff**
Added OpenGL is deprecated; Vulkan support

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -35,28 +35,23 @@ No. Do not post links or downloads to such resources or you will be banned.
...
-To get full speed and compatibility the project will be adopting Vulkan and
-Direct3D 12 so plan accordingly.
+#### Why do you need Vulkan? Won't OpenGL work fine?
 
-#### Why do you need OpenGL 4.5? Won't 3.0 work fine?
-
-Nope. Though technically we could software rasterize everything on a 286 that's
-not a good way to go about things and we make use of the latest tech to get the
-best performance and keep the code sane.
+Nope. OpenGL is depreacted in this project. To get full speed and compatibility the project adapted Vulkan
```
Comment (raynebc) [inline faq.md]: Typo:  Should be "deprecated".

Comment (tmccaff) [inline faq.md]: oops thanks

File: faq.md [hunk 2]
```diff
@@ -35,28 +35,23 @@ No. Do not post links or downloads to such resources or you will be banned.
 
 * Windows 8.1 or Windows 10
 * 64-bit Intel Sandy Bridge or Haswell processor (supporting AVX or AVX2)
-* An OpenGL 4.5-compatible GPU from [this list](http://delphigl.de/glcapsviewer/listreports.php?listreportsbyextension=GL_ARB_bindless_texture&sortby=description_asc) (NVIDIA recommended)
+* An Vulkan compatible GPU from [this list](http://vulkan.gpuinfo.org/) (NVIDIA recommended)
```
Comment (Margen67) [inline faq.md]: *A* Vulkan compatible GPU

Comment (tmccaff) [inline faq.md]: I noticed that. Thanks

File: faq.md [hunk 3]
```diff
@@ -35,28 +35,23 @@ No. Do not post links or downloads to such resources or you will be banned.
...
-Direct3D 12 so plan accordingly.
+#### Why do you need Vulkan? Won't OpenGL work fine?
 
-#### Why do you need OpenGL 4.5? Won't 3.0 work fine?
-
-Nope. Though technically we could software rasterize everything on a 286 that's
-not a good way to go about things and we make use of the latest tech to get the
-best performance and keep the code sane.
+Nope. OpenGL is deprecated in this project. To get full speed and compatibility the project adapted Vulkan
+and Direct3D 12.
```
Comment (tgsm) [inline faq.md]: Xenia doesn't support D3D12, just Vulkan

Comment (tgsm) [inline faq.md]: Xenia does not support D3D12, just Vulkan

Comment (tgsm) [inline faq.md]: Xenia does not support D3D12, just Vulkan

Comment (tmccaff) [inline faq.md]: I thought D3D12 is what the project is aiming for now. Ok I will take that out

**Pull request closed without merging**